Description: Don't use _SOURCES variable without a binary to build from it.
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

--- a/libwnck/Makefile.am
+++ b/libwnck/Makefile.am
@@ -142,14 +142,14 @@
 		--vtail "  return type;\n}\n\n" \
 		$^ > $@
 
-inlinepixbufs_SOURCES = default_icon.png stock_delete.png stock_maximize.png stock_minimize.png
+inlinepixbufs_bitmaps = default_icon.png stock_delete.png stock_maximize.png stock_minimize.png
 inlinepixbufs_pairs =						\
 	default_icon_data $(srcdir)/default_icon.png 		\
 	stock_delete_data $(srcdir)/stock_delete.png 		\
 	stock_maximize_data $(srcdir)/stock_maximize.png 	\
 	stock_minimize_data $(srcdir)/stock_minimize.png
 
-inlinepixbufs.h: $(inlinepixbufs_SOURCES)
+inlinepixbufs.h: $(inlinepixbufs_bitmaps)
 	$(AM_V_GEN)$(GDK_PIXBUF_CSOURCE) --raw --build-list $(inlinepixbufs_pairs) > $@
 
 CLEANFILES =			\
